bittensor/_neuron/text/template_miner/__init__.py

```diff
@@ -397,6 +404,18 @@ def run_epoch( self ):
             wandb.log( { 'stats': wandb.Table( dataframe = df ) }, step = current_block )
             wandb.log( { **wandb_data, **self.dendrite.to_wandb(), **self.axon.to_wandb() }, step = current_block )
 
+    def metagraph_sync(self):
+        r""" Syncing metagraph together with other metagraph-size related objects
+        """
+        self.metagraph.sync()
+
+        if self.moving_avg_scores == None:
+            self.moving_avg_scores = torch.ones_like( self.metagraph.S ) * -1
+
+        if self.metagraph.n > len(self.moving_avg_scores):
+            size_incerease = self.metagraph.n - len(self.moving_avg_scores)
```

Comment:
nit: spelling. size_incerease should be size_increase,

Reply:
Fixed!